README.md

```diff
@@ -1,59 +1,76 @@
+* [X] **GitHub Actions CI/CD**: Preconfigured workflows for automated testing, building, and publishing.
+* [X] **Pixi Package Management**: Integrated with Pixi for efficient and reproducible environment management.
 * [X] **Update Flexibility**: Easily update your project to newer template versions with minimal disruption.
 * [X] **BSD License**: Uses the OSI-approved BSD license, ideal for open-source projects.
 
-For more on pixi, see the [documentation here](https://pixi.sh).
+To install and get started with Pixi, please refer to the [Pixi documentation](https://pixi.sh).
 
 ## Getting Started
 
-### Step 1 - Create from template
+### Step 1 - Create from Template
 
 To create a new Panel extension:
 
 ```bash
 pixi exec --spec copier --spec ruamel.yaml -- copier copy --trust https://github.com/panel-extensions/copier-template-panel-extension <panel-extension-name>
 ```
 
-To update to a newer template version:
+### Step 2 - Create and Add Remote Repository
+
+Go to Github and create your repository with the <panel-extension-name>. Then set the remote:
 
 ```bash
-pixi exec --spec copier --spec ruamel.yaml -- copier update --defaults --trust
+git remote add origin https://github.com/<github-user>/<panel-extension-name>.git
+```
+
+The push your new repository
+
+```bash
+git push --set-upstream origin main
 ```
 
-Note: `copier` will show `Conflict` for files with manual changes during an update—this is normal. As long as there are no merge conflict markers, all patches applied cleanly.
+### Step 3 - Set Up GitHub Pages Docs
 
-### Step 2 - Setup GitHub Pages docs
+Enable GitHub Pages through **Settings > Pages** on the GitHub toolbar:
 
-Enable GitHub Pages through Settings > Pages on the GitHub toolbar:
+<img width="710" alt="image" src="https://private-user-images.githubusercontent.com/42288570/389284308-9bd345e5-7f3f-47c6-80e9-719063396f27.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI0NjE5NTAsIm5iZiI6MTczMjQ2MTY1MCwicGF0aCI6Ii80MjI4ODU3MC8zODkyODQzMDgtOWJkMzQ1ZTUtN2YzZi00N2M2LTgwZTktNzE5MDYzMzk2ZjI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI0VDE1MjA1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2ZTZjM2ZlOGFjMjBjNWM2YmRkZjE4ZjNkMzMyZjczMDVkOGFiMWM3NTA1OTVmZTY4NzE3NTdmZDE1NTM4ODImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rj-hZ-dWRsQUgaE4FYd2H9VBdK7UyBC09zt2P8PAv_c">
 
-<img width="710" alt="image" src="https://github.com/user-attachments/assets/790f4a3e-31ca-42d7-b36e-27e794a504d1">
+On your GitHub repo's **About** section in the sidebar, be sure to add the link to your docs page:
 
-On your GitHub repo's sidebar's About section, be sure to add the link to your docs page and some tags!
+https://<github-user>.github.io/<panel-extension-name>/
```

Comment:
I think there's a button to automatically use the GitHub page.

Reply:
I'm not sure I understand. Could you make the change?